helpers/browser/driver.js

```diff
@@ -61,76 +60,69 @@ class ChromeProtocol {
   }
 
   discardTab() {
-    if (this._instance)
+    if (this._instance) {
       this._instance.close();
+    }
   }
 
   resetFailureTimeout(reject) {
-    if (this.timeoutID)
-        clearTimeout(this.timeoutID);
+    if (this.timeoutID) {
+      clearTimeout(this.timeoutID);
+    }
 
     this.timeoutID = setTimeout(_ => {
       this.discardTab();
       reject(new Error('Trace retrieval timed out'));
-    }, 15 * 1000)
+    }, 15 * 1000);
   }
 
-  subscribeToServiceWorkerDetails(cb, resolve){
+  subscribeToServiceWorkerDetails(cb, resolve) {
     var chrome = this.instance();
 
     chrome.ServiceWorker.enable();
     // chrome.on("ServiceWorker.workerCreated", log)
     // chrome.on("ServiceWorker.workerRegistrationUpdated", log)
-    chrome.on("ServiceWorker.workerVersionUpdated", data => { cb(data, resolve); });
+    chrome.on("ServiceWorker.workerVersionUpdated", data => {
+      cb(data, resolve);
+    });
   }
 
-  evaluateScript(scriptStr){
-    return new Promise((function(resolve, reject) {
-
+  evaluateScript(chrome) { /* TODO(paulirish): scriptStr parameter */
```

Comment:
@paulirish 
should `scriptStr` be passed in the `expression` property below? And this appears to need `chrome` passed in, like `profilePageLoad` below, but this doesn't appear to be called anywhere, so I wasn't sure.